src/store.js

```diff
@@ -256,7 +256,7 @@ function resetStoreVM (store, state, hot) {
   const computed = {}
   forEachValue(wrappedGetters, (fn, key) => {
     // use computed to leverage its lazy-caching mechanism
-    computed[key] = () => fn(store)
+    computed[key] = partial(fn, store)
```

Comment:
Adding some comment to share the context of this memory leak would be nice.

Reply:
Added comment on code